proxy/config_test.go

```diff
@@ -1,12 +1,19 @@
 package proxy
 
 import (
+	"io/ioutil"
 	"testing"
 )
 
 func TestWhenHasErgoFile(t *testing.T) {
 	config := NewConfig()
-	config.Services = LoadServices("../.ergo")
+	services, err := LoadServices("../.ergo")
+	if err != nil {
+		t.Errorf("could not load required configuration file for tests")
+		t.FailNow()
+	}
```

Comment:
You could replace t.Errorf and t.FailNow with a single call to t.Fatal("could not load required configuration file for tests")